cvat/apps/quality_control/quality_reports.py

```diff
@@ -265,6 +265,14 @@ def precision(self) -> float:
     def recall(self) -> float:
         return self.valid_count / (self.gt_count or 1)
 
+    @property
+    def macro_average_precision(self) -> float:
+        return np.average(self.confusion_matrix.precision[: DatasetComparator._UNMATCHED_IDX])
```

Comment:
Please check that 0/0 is not converted to 0, and it doesn't reduce the average quality. It can happen if there are no annotations of some class.